src/vs/workbench/contrib/issue/browser/issueFormService.ts

```diff
@@ -62,14 +70,53 @@ export class IssueFormService implements IIssueFormService {
 		});
 
 	}
-	reloadWithExtensionsDisabled(): Promise<void> {
-		throw new Error('Method not implemented.');
+	async reloadWithExtensionsDisabled(): Promise<void> {
+		if (this.issueReporterWindow) {
+			try {
+				await this.hostService.reload({ disableExtensions: true });
+			} catch (error) {
+				this.logService.error(error);
```

Comment:
what does the user experience when we get here? Do they just see nothing happen?

Reply:
if they click on the help text if they get it, the issue reporter window stays open, and the browser tab is reloaded

I believe with how it is ATM tho, for some reason this help text is permanently hidden  (as of 5 years ago)

<img width="608" alt="Screenshot 2024-07-22 at 12 39 04 PM" src="https://github.com/user-attachments/assets/79251fbf-24db-45f1-af89-a64148089ffb">